### DISCUSSION TITLE
Did mobile version support redirect for Solana Pay?

### DISCUSSION THREAD

**Discussion by katopz** [phantom/docs#84] [General]
Refer to `SPEC1.1` https://github.com/solana-labs/solana-pay/blob/master/SPEC1.1.md#url-describing-a-transfer-request-for-1-sol-with-a-redirect
### URL describing a transfer request for 1 SOL with a redirect
```
solana:mvines9iiHiQTysrwkJjGf2gb9Ex9jXJX8ns3qwf2kN?amount=1&label=Michael&message=Thanks%20for%20all%20the%20fish&memo=OrderId12345&redirect=https%3A%2F%2Fexample.com
```

But only thing I can find is https://docs.phantom.app/phantom-deeplinks/specifying-redirects which is not support.
And I try both `redirect` and `redirect_link` without success so.

1. Did we currently support Solana Pay `redirect`?
2. If not, any plan or timeline to support this.

Thanks

**Comment by adamdelphantom** [comment]
Hey @katopz,

Thank you for writing in!

We don't yet currently support the redirect parameter for Solana Pay. I've made an internal feature request for to support this parameter but I can't provide a timeline for implementation. 

My recommendation would be to keep an eye on our Changelog here: https://github.com/orgs/phantom/discussions/categories/announcements

**Reply by katopz** [reply]
IMHO This `redirect` should save a tons of time to implement/integration and also reduce a number of RPC call behind the scene.
So in case we already have `redirect_link` support for tx already, I think this (must have) addition shouldn't be problem.

Thanks!

**Comment by benan789** [comment]
Hi, was this ever implemented?

**Reply by adamdelphantom** [reply]
@benan789 No this has not been implemented. Unfortunately I do not have an eta to provide.
